### PR TITLE
Major Fixes,

### PR DESCRIPTION
**Pull request by Johnson-Berisha**
also no misspells found.

### CODE REVIEW

**Review by jlooper — CHANGES_REQUESTED** [review]
hi, I'd have to cherry-pick this one. The markdown notations are good (```CSS) but the text isn't quite right. Can you remove the edits to the text and keep the markdown notations please? Then I can accept.

**Comment by Johnson-Berisha** [comment]
accepted

**Comment by Johnson-Berisha** [comment]
Would you want to get rid of the Set the body's font to a given font using the `font-family` element part? (Y/N)

**Comment by jlooper** [comment]
> Would you want to get rid of the Set the body's font to a given font using the `font-family` element part? (Y/N)

Yes, please...let's not make any text changes right now as this will prove tricky for our translators. Thanks!

**Comment by Johnson-Berisha** [comment]
Accepted, its all to you im not a MS worker or a part of your team so what
ever you want.

File: 3-terrarium/2-intro-to-css/README.md [hunk 1]
```diff
@@ -35,6 +36,7 @@ Cascading Style Sheets incorporate the idea that the styles 'cascade' such that
 
 ### Task
 
+This is not a really popular way you should do it.<br>
 Add the inline style "color: red" to your `<h1>` tag:
```
Comment (jlooper) [inline 3-terrarium/2-intro-to-css/README.md:41]: Let's not add this comment, as it's a judgement call I don't want to explain in this section

File: 3-terrarium/2-intro-to-css/README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ In this lesson, we're going to add styles to our online terrarium and learn more
 You should have the HTML for your terrarium built and ready to be styled.
 
 ### Task
+This is how you link you're CSS `style.css` file to you're webpage.<br>
```
Comment (jlooper) [inline 3-terrarium/2-intro-to-css/README.md:24]: We don't need this explanation here, also please beware of spelling errors, thanks

File: 3-terrarium/2-intro-to-css/README.md [hunk 3]
```diff
@@ -59,9 +61,9 @@ Styles are inherited from an ancestor style to a descendent, such that nested el
 
 ### Task
 
-Set the body's font to a given font, and check to see a nested element's font:
+Set the body's font to a given font using the `font-family` element, and check to see a nested element's font:
```
Comment (jlooper) [inline 3-terrarium/2-intro-to-css/README.md:65]: Let's not do this as it's explicit below

File: 3-terrarium/2-intro-to-css/README.md [hunk 4]
```diff
@@ -94,7 +96,7 @@ h1 {
 
 This way of styling a tag gives you control over unique elements, but you need to control the styles of many plants in your terrarium. To do that, you need to leverage CSS selectors.
 
-### Ids
+### Id's
```
Comment (jlooper) [inline 3-terrarium/2-intro-to-css/README.md:100]: This is not a possessive word, but rather a plural, so we don't need the apostrophe

**Pull request closed without merging**